src/lib/CONFIG/config.h

```diff
@@ -14,7 +14,7 @@
 #define TX_CONFIG_MAGIC     (0b01U << 30)
 #define RX_CONFIG_MAGIC     (0b10U << 30)
 
-#define TX_CONFIG_VERSION   7U
+#define TX_CONFIG_VERSION   8U
```

Comment:
Why does the config version need to change when the struct size stays the same and is already initialized to 0? This is why we allocated the bytes for the buttons in the config PR.

Reply:
It probably doesn't need to be updated. I'll check and remove if it's not required.

Reply:
Yup, not needed, removed!